src/dlgprefreplaygaindlg.ui

```diff
@@ -209,6 +255,19 @@
         </property>
        </widget>
       </item>
+      <item row="0" column="0">
+       <widget class="QLabel" name="label_8">
+        <property name="text">
+         <string>The normal average ReplayGain playback level is -14 dB. This offers enough headroom for the dynamic peaks of almost all tracks. However it might be not enough for mixing or adding effects without clipping or it might sound to quiet for modern pop music that is normalized to 0 db and compressed to an average playback level of -7 dB or more.  So you may tweak the Initial Boost to solve these issues.</string>
```

Comment:
The normal average ReplayGain playback level is -14 dB. This offers enough headroom for the dynamic peaks of almost all tracks. However it might be not enough for mixing or adding effects without clipping or it might sound to quiet for modern pop music that is normalized to 0 db and compressed to an average playback level of -7 dB or more.  So you may tweak the Initial Boost to solve these issues.